source/vecs/entitymanager.d

```diff
@@ -1109,7 +1140,13 @@ private:
 	assert(*integral == 0);
 	assert(*str == "Hello");
 
-	entity.emplace!int(45);
```

Comment:
This line was deleted. Is this still covered or just accidental?

Reply:
It's covered. That line was using `emplace` to cover an update. Now I changed it to `patch`.

Reply:
This also plays out with what I want to do in the future. Having `add, emplace and set` only to **add** an entity and component to a Storage, and having `patch and <some_other_function>` to update the components of existing entities in a Storage. Then I will add functions that behave like them at the moment (if an entity exists update, otherwise insert).